backend/src/manager/sync.rs

```diff
@@ -41,7 +44,7 @@ async fn synchronize_once(shared: &ManagerSharedState) -> Result<Status, Error>
             MainStatus::BackingUp { .. } => (),
         },
         Status::Starting => match *status {
-            MainStatus::Stopped | MainStatus::Stopping => {
+            MainStatus::Stopped | MainStatus::Stopping | MainStatus::Restarting => {
```

Comment:
are these all potential cases of from -> to states?

Reply:
kind of. The `Status` states are the state of the Manager, the `MainStatus` is the desired state of the package.